base/docs/helpdb/Base.jl

```diff
@@ -9706,21 +9706,21 @@ modified by the current file creation mask.
 mkdir
 
 """
-    bytestring(::Ptr{UInt8}, [length])
```

Comment:
Maybe time to move these docs inline and put them next to the other methods for `String`?